src/overlays/actors/ovl_En_Nimotsu/z_en_nimotsu.h

```diff
@@ -7,8 +7,11 @@ struct EnNimotsu;
 
 typedef struct EnNimotsu {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x98];
-} EnNimotsu; // size = 0x1DC
+    /* 0x144 */ char unk_144[0x48];
```

Comment:
```suggestion
    /* 0x144 */ UNK_TYPE1 unk_144[0x48];
```
`UNK_TYPE1` is preferred as the type for unknown struct padding.

Reply:
Addressed in commit 137a1c8